src/main/java/jdbchomework/utils/HibernateUtil.java

```diff
@@ -1,17 +1,32 @@
 package jdbchomework.utils;
 
+import org.hibernate.HibernateException;
 import org.hibernate.SessionFactory;
 import org.hibernate.cfg.Configuration;
+import org.slf4j.LoggerFactory;
 
 public class HibernateUtil {
+    private static org.slf4j.Logger log = LoggerFactory.getLogger(HibernateUtil.class);
+    private static SessionFactory sessionFactory;
 
-    private static SessionFactory sessionFactory = new Configuration().configure().buildSessionFactory();
+    static {
+        try {
+            sessionFactory = new Configuration().configure().buildSessionFactory();
+        } catch (HibernateException e) {
+            throw new RuntimeException("Cannot create Session Factory", e);
+        }
+    }
 
     public static SessionFactory getSessionFactory() {
         return sessionFactory;
     }
 
     public static void closeSessionFactory() {
-        sessionFactory.close();
+        try {
+            sessionFactory.close();
+        } catch (HibernateException e) {
+            log.error("Cannot close sessionFactory", e);
+            throw new RuntimeException("Cannot close sessionFactory", e);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00112)